src/sections/service.html

```diff
@@ -0,0 +1,51 @@
+
+<!-- section service -->
+<!-- <div class="grid md:grid-cols-2 grid-rows-1 border border-red-600 px-8 my-16 md:px-16 lg:px-24 gap-10">
+    <div class="space-y-8 font-body lg:text-5xl  md:text-4xl  text-2xl font-medium">
+    <p>Brand strategy</p>
+    <p>Corporate Identity & <br> Website development</p>
+    <p>SEO & copywriting</p>
+    <p>Media Relations</p>
+    <p>Digital marketing & <br> content production</p>
+    <p>Influencer relations</p>
+    </div>
+    <-- green vector -->  <!-- vector -->
+    <!-- <div class="px-16 relative bg-no-repeat bg-cover  md:px-16 lg:px-24 bg-center md:bg-contain bg-[url('../../img/Vector-5.webp')] md:text-white text-xl font-body">
+    
+        <div class="md:absolute w-72 text-2xl  mb-10     md:w-1/3 md:left-96 lg:pr-8 lg:mb-2  md:top-24  ">
+            <p class="text-white pt-24 md:py-0 w-full">Our strategic approach is a collaborative journey that starts with analysing the initial situation and leads to messages 
+                that cut through the information overload. We take care of content production and maintain relationships with 
+                possible way.we accompany you 
+                every step of the way, ensuring that every measure is effective.
+            </p>
+    
+        </div>
+        
+    </div>
+    </div> --> 
+    <section class="px-[2rem] md:pt-36 md:grid md:grid-cols-2 md:grid-rows-1 md:justify-end md:gap-48 md:pl-24 md:pr-[10.5rem] md:min-h-[48.75rem] md:min-w-[71.973rem] md:pb-72 pb-72">
+        <div class="md:col-span-1 md:row-span-1 space-y-16 font-body lg:text-5xl  md:text-4xl text-2xl font-medium">
+            <p>Brand strategy</p>
+            <p>Corporate Identity & <br> Website development</p>
+            <p>SEO & copywriting</p>
+            <p>Media Relations</p>
+            <p>Digital marketing & <br> content production</p>
+            <p>Influencer relations</p>
+        </div>
+        <div class="  md:col-span-1  md:row-span-1 bg-[url('../../img/Vector-5.webp')] md:w-[29.285rem] h-[100%] md:bg-contain bg-contain bg-no-repeat md:mt-0 mt-16 ">
```

Comment:
There is an extra white space between classes. May you remove it? Please consider not using arbitrary(%) values as requested in the morning stand-up.